static-build: qemu5: update qemu build dependecies

Add libselinux1-dev, libffi-dev and libmount-dev as build dependencies  to fix the following error

```
16:11:15 /usr/bin/ld: cannot find -lselinux
16:11:15 /usr/bin/ld: cannot find -lmount
16:11:15 /usr/lib/gcc/x86_64-linux-gnu/7/../../../x86_64-linux-gnu/libgmodule-2.0.a(libgmodule_2_0_la-gmodule.o): In function `g_module_open':
16:11:15 (.text+0x4f5): warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
16:11:15 /usr/bin/ld: cannot find -lffi
16:11:15 /usr/bin/ld: cannot find -lselinux
16:11:15 /usr/bin/ld: cannot find -lmount
16:11:15 /usr/bin/ld: cannot find -lffi
16:11:15 collect2: error: ld returned 1 exit status
16:11:15 make[1]: *** [qemu-system-x86_64] Error 1
16:11:15 Makefile:208: recipe for target 'qemu-system-x86_64' failed
16:11:15 make: *** [x86_64-softmmu/all] Error 2
16:11:15 Makefile:527: recipe for target 'x86_64-soft
```

--- a/static-build/qemu/Dockerfile
+++ b/static-build/qemu/Dockerfile
@@ -23,12 +23,15 @@ RUN apt-get --no-install-recommends install -y \
 	    libcap-ng-dev \
 	    libdw-dev \
 	    libelf-dev \
+	    libffi-dev \
 	    libglib2.0-0 \
 	    libglib2.0-dev \
 	    libglib2.0-dev git \
 	    libltdl-dev \
+	    libmount-dev \
 	    libpixman-1-dev \
 	    libpmem-dev \
+	    libselinux1-dev \
 	    libtool \
 	    pkg-config \
 	    pkg-config \

